tests/fitting_test.py

```diff
@@ -304,7 +304,7 @@ def test_with_init(self, sim_high_stat):
             fitter.param_unc("bad_name")
         if "gauss_amp" in fitter.param_names:
             u = fitter.param_rel_unc("gauss_amp")
-            assert u is None or u < 0.01
+            assert u is None or (u < 0.01 and u >= 0)
```

Comment:
What about `(0 <= u < 0.01)`?

Reply:
I usually avoid that because it doesn't work for numpy arrays (although we're not dealing with an array here)